app/views/configure/reading.phtml

```diff
@@ -135,6 +135,16 @@
 					</select>
 				</div>
 			</div>
+			<div class="form-group">
+				<label class="group-name" for="show_icons"><?= _t('conf.reading.article.icons') ?></label>
+				<div class="group-controls">
+					<select name="show_icons" id="show_icons" data-leave-validation="<?= FreshRSS_Context::userConf()->show_icons ?>">
+						<option value="a" <?= FreshRSS_Context::userConf()->show_icons === 'a' ? ' selected="selected"' : '' ?> data-input-visible="true"><?= _t('conf.reading.article.icons.with_authors') ?></option>
+						<option value="h" <?= FreshRSS_Context::userConf()->show_icons === 'h' ? ' selected="selected"' : '' ?> data-input-visible="true"><?= _t('conf.reading.article.icons.header') ?></option>
+						<option value="t" <?= FreshRSS_Context::userConf()->show_icons === 't' ? ' selected="selected"' : '' ?> data-input-visible="true"><?= _t('conf.reading.article.icons.above_title') ?></option>
```

Comment:
I prefer to use more readable value keys (a = above, t = title etc.)

Reply:
I copied this from the existing settings that use this approach. I also would have preferred these settings to be meaningful.